src/test/java/com/solvd/SearchTest.java

```diff
@@ -3,41 +3,24 @@
 import com.solvd.pages.HomePage;
 import com.solvd.pages.ResultPage;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.Assert;
-import org.testng.annotations.AfterTest;
-import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
-import java.util.concurrent.TimeUnit;
-
-public class SearchTest {
-    private final String driverPath = "/Users/yanagalitsyna/Desktop/chromedrivers/chromedriver";
-    private WebDriver driver;
-    private HomePage homePage;
-    private ResultPage resultPage;
-
-    @BeforeTest
-    public void setup(){
-        System.setProperty("webdriver.chrome.driver", driverPath);
-        driver = new ChromeDriver();
-        driver.manage().timeouts().pageLoadTimeout(10, TimeUnit.SECONDS);
-        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
-        driver.get("https://www.amazon.com/");
+public class SearchTest extends AbstractTest{
 
+    @Test
+    public void testLogo(){
+        WebDriver driver = driverThreadLocal.get();
+        HomePage homePage = new HomePage(driver);
```

Comment:
You repeat lines with setting driver and homepage in both tests so may be you can move them to @BeforeMethod